src/test/groovy/org/gradle/profiler/ProfilerIntegrationTest.groovy

```diff
@@ -459,7 +459,7 @@ plugins.withId("idea") {
         def lines = resultFile.lines
         lines.size() == totalLinesForExecutions(16)
         lines.get(0) == "scenario,ideaModel,ideaModel"
-        lines.get(1) == "version,$minimalSupportedGradleVersion,$latestSupportedGradleVersion"
+        lines.get(1) == "version,Gradle $minimalSupportedGradleVersion,Gradle $latestSupportedGradleVersion"
```

Comment:
I find it strange the the `version` in the CSV contains `Gradle`, I would expect it to be the version only. What happens for Maven, is the version then just `Maven`?

Reply:
So before this change we had `maven`, `bazel` and `6.7` (representing Gradle). Now we have `maven`, `bazel` and `Gradle 6.7`. I don't think it's ideal, but it's somewhat better for our purposes, and not worse in any way compared to what we had before.

I'd like to keep focus on useful improvements. If we find that Gradle vs Maven or Buck or Bazel has problems, I think it's fine to address them then.

Reply:
It is fine for now.